std_msgs/Header header
bool manual_speed_control ##手动速度控制
bool manual_steer_control ##手动方向盘转角控制
bool stop_cmd             ##手动停车指令
float64 v_r               ##目标速度
float64 steer_r           ##目标方向盘转角
float64 v_r_1             ##上一时刻目标速度
